src/components/Chart.jsx

```diff
@@ -50,6 +50,7 @@ class Chart extends Component {
 
     render() {
         const {
+            chartId, // eslint-disable-line no-unused-vars
```

Comment:
why did you disable this?

Reply:
@brucebinary  its not being used directly in chart.jsx so eslit creates a error.
`  53:13  error  'chartId' is assigned a value but never used  no-unused-vars`

Reply:
if it not being used why do you include it?

Reply:
@brucebinary Its being used in updateProps in chartstate.js.

Reply:
@brucebinary yea. i realy dont need it!